.travis.yml

```diff
@@ -1,21 +1,18 @@
 language: python
 
 python:
-  - "3.4"
   - "3.5"
+  - "3.6"
```

Comment:
Sorry, I'm typing from phone, I've fixed a few typos above.
It adds separate job for 3.7, which will not influence build outcome, this is a related change.
Also it removes deployment from test jobs in favor of a separate one running after all test jobs. This could be done in a separate PR, yes.

Reply:
That's impressive, typing Travis configurations from memory. :)

I'll submit a separate PR with your suggestions soon.

Reply:
Haha 😂. That's because I do this for the whole aio-libs, trying to come up with an auto-generated solution for configs globally :)
Oh, and also I'm lazy to pull myself up of the bed :)